src/process_utility.c

```diff
@@ -269,6 +269,46 @@ reindex_chunk(Oid chunk_relid, void *arg)
 	}
 }
 
+static bool
+process_drop(Node *parsetree)
```

Comment:
How do we deal with `CASCADE` here? If I do `DROP TABLE foo;` (without `CASCADE`) should we drop the hypertable and all chunks or should we require `CASCADE`? I think probably we should check for cascade to retain the semantics of regular postgres. This also gives some extra safety.

Reply:
Here's another example of unclear semantics. Let's say there's an object that depends on a hypertable, e.g., there is a foreign key reference to a hypertable (or chunk row, I guess). Since this function does not check for `CASCADE` and no such information is passed on in the upcall, how will this cascade to removing the referencing object? I guess what happens now is that this simply fails if some external object depends on the hypertable? Overall, it is really unclear to me what the semantics are here since no cascade information is passed in the upcall.

Reply:
I think we made a decision here https://iobeam.slack.com/archives/C2XABL4CC/p1496926029038257.